test/test_prelude.py

```diff
@@ -0,0 +1,90 @@
+# coding: utf-8
+
+from pathlib import Path
+import shutil
+import requests
+from zipfile import ZipFile
+import os
+import glob
+import nibabel as nib
+import numpy as np
+
+import pytest
+
+from shimmingtoolbox.unwrap import prelude
+
+
+class TestCore(object):
+
+    def setup(self):
```

Comment:
A good practive for unit tests is to have every test be completly independent. Then, you have to make sure that your setup and teardown is called before each test that you will have in this class (if you have more than one)

Reply:
The actual name to run a setup and a teardown before each test method is `setup_method` and `teardown_method` for pytest